CODEOWNERS

```diff
@@ -4,11 +4,17 @@
 # Covers all web content
 /src/       @sfshaza2 @atsansone @parlough
 
+# Covers Jekyll plugins
+/src/_plugins/ @khanhnwin @parlough @sfshaza2 @atsansone
+
+# Covers website styles
+/src/_sass/ @khanhnwin @parlough @sfshaza2 @atsansone
+
 # Covers website tooling
 /tool/      @atsansone @parlough
 
 # Covers examples in files
 /examples/  @domesticmouse @johnpryan @khanhnwin @parlough
 
 # Covers all configuration files for the website
-/*          @atsansone @parlough
+/*          @atsansone @parlough @sfshaza2
```

Comment:
this would include Dockerfile and everything in the root directory? I think I should also be added here?